.gitignore

```diff
@@ -27,3 +27,6 @@ target/
 build/
 !**/src/main/**/build/
 !**/src/test/**/build/
+
+### Custom ###
+*.env
```

Comment:
Custom .env files are usually called .env.local, .env.test, etc. So the pattern should be .env.*